DependencyInjection/CmfCoreExtension.php

```diff
@@ -128,7 +128,6 @@ public function prepend(ContainerBuilder $container)
                                 'persistence' => array(
                                     'phpcr' => array(
                                         'enabled' => $persistenceConfig['enabled'],
-                                        'use_sonata_admin' => $persistenceConfig['use_sonata_admin'],
```

Comment:
this change makes us ignore use_sonata_admin. we also need to change the configuration to no longer allow this flag...